src/main/screen_pause.cpp

```diff
@@ -157,7 +157,7 @@ static bool s_QuitTesting()
 
 static bool s_SaveAndContinue()
 {
-    bool CanSave = (LevelSelect || (IsEpisodeIntro && NoMap)) && !Cheater;
+    bool CanSave = (LevelSelect || (IsEpisodeIntro && NoMap) || IsHubLevel) && !Cheater;
```

Comment:
Replace with simply `(LevelSelect || IsHubLevel)` (and, scan the whole repo for all other cases of `IsEpisodeIntro && NoMap`)

Reply:
Done, I replaced all such cases as `IsHubLevel` replaces that condition.